sdk/src/Core/Amazon.Runtime/EventStreams/Internal/EventOutputStream.cs

```diff
@@ -20,20 +20,17 @@
 using System.Threading;
 using ThirdParty.RuntimeBackports;
 
-#if AWS_ASYNC_API
+
 using System.Threading.Tasks;
-#else
-using System.Threading;
-#endif
 
 namespace Amazon.Runtime.EventStreams.Internal
 {
     /// <summary>
-    /// The contract for the <see cref="EventStream{T,TE}"/>.
+    /// The contract for the <see cref="EventOutputStream{T,TE}"/>.
     /// </summary>
     /// <typeparam name="T">An implementation of IEventStreamEvent (e.g. IS3Event).</typeparam>
     /// <typeparam name="TE">An implementation of EventStreamException (e.g. S3EventStreamException).</typeparam>
-    public interface IEventStream<T, TE> : IDisposable where T : IEventStreamEvent where TE : EventStreamException, new()
+    public interface IEventOutputStream<T, TE> : IDisposable where T : IEventStreamEvent where TE : EventStreamException, new()
```

Comment:
I renamed this to `IEventOutputStream` because this stream is only used for events that come from the service. I don't want users to be confused this is a stream that could be used for input.